### PR TITLE
look refinement plus extra features

### PR DESCRIPTION
**Pull request by vonovak**

### CODE REVIEW

File: lib/Toolbar.js [hunk 1]
```diff
@@ -56,7 +56,6 @@ export default class Toolbar extends Component {
         actions: PropTypes.arrayOf(PropTypes.shape({
             icon: PropTypes.string.isRequired,
             onPress: PropTypes.func,
-            counter: PropTypes.shape()
```
Comment (xotahal) [inline lib/Toolbar.js]: Why did you remove this line?

Comment (vonovak) [inline lib/Toolbar.js]: Seems like it wasn't used anywhere in the file

Comment (xotahal) [inline lib/Toolbar.js]: You're right. Need to refactor this library :/ Thank you Vojta, I'll merge it ;) 👍

**Comment by vonovak** [comment]
so, I wanted to use this for the search field but now I noticed the default `ToolbarAndroid` can do that too. just fyi
